main.go

```diff
@@ -35,12 +37,10 @@ func main() {
 		Help: "The total number of errors per RPC metric collector.",
 	}, []string{"collector"})
 
-	timeout := 600 * time.Millisecond
-
 	reg := prometheus.NewRegistry()
-	reg.Register(prometheus.NewProcessCollector(os.Getpid(), ""))
+	reg.Register(prometheus.NewProcessCollector(prometheus.ProcessCollectorOpts{}))
```

Comment:
The new client_golang version required for this change hasn't been included in the vendoring yet.